### PR TITLE
Add getting started GIF for intellij too

### PR DESCRIPTION
**Pull request by etanshaul**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,9 +9,14 @@ Spend less time learning about configuration and management of your app - and sp
 ## What's in this repo
 
 Code templates for easy Getting-Started experience with Google Cloud Code in Python, Java, Nodejs, Go and .NET Core.
-We support two IDEs: Visual Studio Code and IntelliJ.
+We support two IDEs: Visual Studio Code and IntelliJ (and other JetBrains IDEs).
```
Comment (etanshaul) [inline README.md]: adding a small wording update in here too.

**Comment by etanshaul** [comment]
> Great idea, thanks for the effort. These GIFs are hard!

All the thanks goes to @sanguinecloud . I am simply reusing the same gif already on our quickstart!

File: README.md (hunk 1 above)
Comment (simonz130) [reply README.md]: Do you support Rider for .NET?

Comment (etanshaul) [reply README.md]: No, not currently

Comment (etanshaul) [reply README.md]: No, not currently

**Pull request merged**